[SP-19] некоторые изменения

DarkAngel77792 opened the pull request:
добавил изменения в задаче и переместил остальные решенные задачи в папку other

DarkAngel77792 committed:
[SP-19] привет

amatyushov commented on dshpilka/other/src/main.cpp:
```diff
@@ -0,0 +1,295 @@
+#include <cmath>
+
+/// <summery>
+/// ���� ������������ ����� - ���� 1 �� ������. ������� ��������� 0.1, 0.2, ..., 1 �� ������.
```
Исправь кодировку файла при сохранении

Tools > Options > Environment > Documents > Save documents as Unicode when data cannot be saved in codepage

 и перезалей его

amatyushov commented on dshpilka/sp-19/src/main.cpp:
```diff
@@ -42,12 +33,13 @@ int main()
+
+int main()
+{
+	//task_palindrome();
```
раскоментировать

amatyushov commented on dshpilka/other/src/main.cpp:
```diff
@@ -0,0 +1,295 @@
+	//task_recursion();
+	task_recursion2();
+	//task_recursion3()
+	//task_recursion4();
```
раскоментировать все строки

DarkAngel77792 committed:
[SP-19] раскоментировал строку

DarkAngel77792 committed:
[SP-19] Update file codepage

DarkAngel77792 committed:
[SP-19] task_recursion_chek_pow2

добавил решение задачи на определение является ли введенное число точной степенью двойки с помощью рекурсии

DarkAngel77792 committed:
[SP-19]task_recursion_from_right_to_left

добавил решение задачи, которая принимает на ввод число и с помощью рекурсии выводит его через пробел справа налево

DarkAngel77792 committed:
[SP-19]-save-other-tasks-and-sum-changes

Добавил решение 2ух задач на рекурсию

DarkAngel77792 committed:
Merge branch 'develop' into feature/SP-19-save-other-tasks-and-sum-changes
